localBuildRun.sh

```diff
@@ -3,4 +3,8 @@ cp engine/parser.def build/engine/parser.def
 cp -r ../tutorials build/playbooks
 cp -r environments build
 cp -r runners build
+npm test
+if [ $? -eq 1 ]; then 
+    echo 'tests failed' 
+fi
```

Comment:
In the powershell script you throw an error when tests will fail. This ends the task. Here you just log it to the console and the task continues. Maybe its better to exit here as well to keep it consistent